Adding {targets} architecture, copying code from NW-CLP-RS (#4)
rossyndicate/Landsat_C2_SR_GLORIA: README.md

# Landsat_C2_SR_GLORIA

Code to pull, collate, and correct Landsat SR product at GLORIA and other sites as presented in Maciel et al. 2023 (<https://doi.org/10.1002/lol2.10344>).

This repository is covered by the MIT use license. We request that all downstream uses of this work be available to the public when possible.

## Code Architecture

> This repository and analysis uses the {targets} package to create intelligent workflows. Because the GEE pull was run before adding additional {targets} groups, the functions are run in the file `b_data_acquisition.R`, but the source/in/out folders are in a folder that does not contain an indication of the targets group ('b' in this case). Changing the folder name from `data_acquisition` would invalidate that portion of the {targets} workflow, requiring a re-pull of the GEE data to become up to date.

Reviewer: ```suggestion
This repository and analysis uses the {targets} package to create intelligent workflows. Because the GEE pull was run before adding additional {targets} groups, the functions are run in the file `b_data_acquisition.R`, but the 'src', 'in', and 'out' folders are in a folder that does not contain an indication of the targets group (i.e., 'data acquisition', without 'b' in this case). Changing the folder name from `data_acquisition` would invalidate that portion of the {targets} workflow, requiring a re-pull of the GEE data to become up to date.
```
